### ISSUE TITLE
Add possibility to show different/multiple insta feeds

### ISSUE THREAD

**Issue by wm-simon** [austenc/statamic-instagram-feed#7]

**Comment by austenc** [comment]
Thanks @Pfeifakopf! 👍

**Comment by austenc** [comment]
@Pfeifakopf This feature is now available (in version `2.2.0`), thank you for the suggestion!

Simply pass in a `profile` parameter to the tag when you use it.

```
{{ instagram_feed profile="austencam" }}
    ...
{{ /instagram_feed }}
```
